app/common/lib/bookmarkUtil.js

```diff
@@ -71,57 +83,94 @@ let lastWidth
 const getToolbarBookmarks = (state) => {
   const sites = state.get('sites', Immutable.List())
   const windowWidth = window.innerWidth
-  if (sites === oldSites &&
-      lastWidth === windowWidth &&
-      lastValue) {
+
+  if (sites === oldSites && lastWidth === windowWidth && lastValue) {
     return lastValue
   }
+
   oldSites = sites
   lastWidth = windowWidth
 
   const noParentItems = siteUtil.getBookmarks(sites)
     .filter((bookmark) => !bookmark.get('parentFolderId'))
     .sort(siteUtil.siteSort)
+
   let widthAccountedFor = 0
-  const overflowButtonWidth = 25
+
+  const onlyText = showOnlyText()
+  const textAndFavicon = showTextAndFavicon()
   const onlyFavicon = showOnlyFavicon()
-  const favicon = showFavicon()
 
-  // Dynamically calculate how many bookmark items should appear on the toolbar
-  // before it is actually rendered.
-  const maxWidth = domUtil.getStyleConstants('bookmark-item-max-width')
-  const padding = domUtil.getStyleConstants('bookmark-item-padding') * 2
-  // Toolbar padding is only on the left
-  const toolbarPadding = domUtil.getStyleConstants('bookmarks-toolbar-padding')
-  const bookmarkItemMargin = domUtil.getStyleConstants('bookmark-item-margin') * 2
+  const maxWidth = parseInt(globalStyles.spacing.bookmarksItemMaxWidth, 10)
+  const padding = parseInt(globalStyles.spacing.bookmarksItemPadding, 10) * 2
+  const bookmarkItemMargin = parseInt(globalStyles.spacing.bookmarksItemMargin, 10) * 2
+  const fontSize = parseInt(globalStyles.spacing.bookmarksItemFontSize)
+  const fontFamily = globalStyles.defaultFontFamily
+  const chevronMargin = parseInt(globalStyles.spacing.bookmarksItemChevronMargin)
+  const chevronFontSize = parseInt(globalStyles.spacing.bookmarksItemChevronFontSize)
+  const chevronWidth = chevronMargin + chevronFontSize
+
   // No margin for show only favicons
-  const chevronMargin = domUtil.getStyleConstants('bookmark-item-chevron-margin')
-  const fontSize = domUtil.getStyleConstants('bookmark-item-font-size')
-  const fontFamily = domUtil.getStyleConstants('default-font-family')
-  const chevronWidth = chevronMargin + fontSize
-  const margin = favicon && onlyFavicon ? 0 : bookmarkItemMargin
+  const margin = onlyFavicon ? 0 : bookmarkItemMargin
+
+  // Toolbar padding is only on the left
+  const toolbarPadding = parseInt(globalStyles.spacing.bookmarksToolbarPadding)
+
+  const overflowButtonWidth = 25
```

Comment:
I know it was like this before but could you please add this to `globalStyles` as well?